Makefile

```diff
@@ -69,11 +69,6 @@ test-agent-%: venv
 test-compose: venv
 	pytest scripts/tests/*_tests.py -v -s $(JUNIT_OPT)/compose-junit.xml
 
-test-compose-2:
```

Comment:
why remove this?  yes python2 is EOL but so many system still ship with it, if possible would be nice to keep compat for stop/start

Reply:
we could keep it for a while